src/api/runtimeApi.ts

```diff
@@ -26,19 +27,24 @@ export async function getBalance(walletEncoded: any) {
   fix getUnboundong in wsProvider
   use Ws
 */
-export async function getUnboundOxg(nodeAddress: string, ssl: boolean, walletEncoded: any) {
-  const wallet = getWallet(walletEncoded);
+export async function getUnboundOxg( walletEncoded: any) {
+	const wallet = getWallet(walletEncoded);
   const address = getAccount(wallet).address;
-  const protocol = ssl ? "https" : "http";
-  const baseUrl = `${protocol}://${nodeAddress}:${CONST.HTTP_REST_PORT}`;
-  const unboundOngUrl = "/api/v1/unboundoxg/";
+  const unboundOngUrl = `/addresses/${address.toBase58()}/balances`;
```

Comment:
If only unboundoxg is needed, then it will be better for performance to directly use blockchain REST API (example https://andromeda-sync.onyxpay.co:20334/api/v1/unboundoxg/AdEYNwzKkJgje6P6cBRcwn7qoh2XKThi5q )